src/schema/event.ts

```diff
@@ -2,18 +2,26 @@ import { z } from "zod";
 
 export const createEventSchema = z
   .object({
-    name: z.string().min(1, { message: "Event name is required" }),
-    location: z.string().min(1, { message: "Event location is required" }),
-    description: z.string().min(5, {
-      message: "Description of the event must be more than 5 characters long."
-    }),
+    name: z
+      .string({ required_error: "Event name is required" })
+      .min(1, { message: "Event name must be 1 or more characters long" }),
```

Comment:
”min(1, { message: "Event name must be 1 or more characters long" })“ This line has no effect right now and can be removed.